earthfile2llb/interpreter.go

```diff
@@ -369,38 +359,34 @@ func (i *Interpreter) handleFrom(ctx context.Context, cmd spec.Command) error {
 	if i.pushOnlyAllowed {
 		return i.pushOnlyErr(cmd.SourceLocation)
 	}
-	fs := flag.NewFlagSet("FROM", flag.ContinueOnError)
-	allowPrivilegedFlag := fs.Bool("allow-privileged", false, "Enable privileged mode")
-	buildArgs := new(StringSliceFlag)
-	fs.Var(buildArgs, "build-arg", "A build arg override passed on to a referenced Earthly target")
-	platformStr := fs.String("platform", "", "The platform to use")
-	err := fs.Parse(getArgsCopy(cmd))
+	opts := fromOpts{}
+	args, err := parseArgs("RUN", &opts, getArgsCopy(cmd))
```

Comment:
Should this be `FROM`?

```suggestion
	args, err := parseArgs("FROM", &opts, getArgsCopy(cmd))
```

Reply:
good catch!